visualization/app/codeCharta/ui/edgeChooser/edgeChooser.component.html

```diff
@@ -14,15 +14,18 @@
 		ng-disabled="$ctrl.noEdgesAvailable()"
 	>
 		<md-select-header>
-			<input
-				id="edge-metric-selector"
-				class="metric-search"
-				ng-model="$ctrl._viewModel.searchTerm"
-				type="text"
-				placeholder="&#xF002; Metric (highest value)"
-				ng-keydown="$event.stopPropagation()"
-				ng-change="$ctrl.filterMetricData()"
-			/>
+			<div>
```

Comment:
Classless div, is it needed here? Putting all the childs of this div directly into md-select-header would have the same result